bin/tessel-2.js

```diff
@@ -19,9 +19,12 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
 
 // Check for updates
 const pkg = require('../package.json');
-updateNotifier({
-  pkg
-}).notify();
+
+try {
+  updateNotifier({ pkg }).notify();
+} catch (err) {
+  CrashReporter.submit(err.stack);
```

Comment:
It shouldn't because the error is caught. `CrashReporter.submit` doesn't exit the process.